components/AllowList/AllowList.tsx

```diff
@@ -3,10 +3,13 @@ import AllowListForm from "../AllowListForm"
 
 const AllowList = () => {
   const [twitterHandle, setTwitterHandle] = useState("")
+  const [publicWallet, setPublicWallet] = useState("")
```

Comment:
same question, is there any reason state is being preserved here instead of in the child component? 

Looks unused here right?